backend/estimate-service/src/main/java/org/egov/works/config/EstimateServiceConfiguration.java

```diff
@@ -70,5 +71,24 @@ public void initialize() {
 
     @Value("${estimate.search.max.limit}")
     private Integer maxLimit;
+
+    //Workflow Config
+    @Value("${pgr.business.codes}")
+    private List<String> businessServiceList;
+
+    @Value("${egov.workflow.host}")
+    private String wfHost;
+
+    @Value("${egov.workflow.transition.path}")
+    private String wfTransitionPath;
+
+    @Value("${egov.workflow.businessservice.search.path}")
+    private String wfBusinessServiceSearchPath;
+
+    @Value("${egov.workflow.processinstance.search.path}")
+    private String wfProcessInstanceSearchPath;
+
+    @Value("${is.workflow.enabled}")
+    private Boolean isWorkflowEnabled;
```

Comment:
What if it is disabled?
I think this property existed in those services because they had logic in case when the workflow service didn't exist. 
We are not supporting such a scenario.

Reply:
ohhok. I"ll remove this property.